glue/index.js

```diff
@@ -17,12 +17,15 @@ export class Asdom {
 	_exports = null
 
 	wasmImports = {
-		asdomImports: {
```

Comment:
Just nitpicking here, but "asdom" instead of "asDOM" would be better so it matches with the usages elsewhere. But we could also change the other usages. Do you like asDOM better?